packages/client/wallets/smart-wallet/src/evm/transfer.ts

```diff
@@ -0,0 +1,83 @@
+import { type Abi, type Address, erc20Abi, erc721Abi } from "viem";
+
+import erc1155Abi from "@/abi/erc1155";
+
+import type { SmartWalletChain } from "./chains";
+
+interface EVMToken {
```

Comment:
same here, would move this out to a specific types file

Reply:
https://github.com/Crossmint/crossmint-sdk/pull/913/commits/bbe1a5f6c4c5b9b2014937bd852be8095e061d26